static/flow/views/program-editor-panel.js

```diff
@@ -305,7 +303,7 @@ var ProgramEditorPanel = function(options) {
         // Add menu
         //
         var menuData = createMenuData();
-        if (!_this.isFilterBlock(block.type) && !_this.isDeviceBlock(block.type) && block.type!="plot") {
+        if (!_this.isFilterBlock(block.type) && !_this.isDeviceBlock(block.type) && block.type!="plot" && block.type!="timer") {
```

Comment:
@dstrawberrygirl For now, the model does not allow a sensor or timer block to be renamed because a user might give the block a name that conflicts with the flow client's assumption about block names.  So if a user renames a timer to have the name "light", the timer block might start getting light sensor values!  This design is less than ideal. 
 In a perfect world, user-facing block names should not be used as unique identifiers for blocks.  But without the time needed to rework this model in both the flow client and flow-server, this existing model will have to work for now.